.ci/packaging.groovy

```diff
@@ -149,74 +159,174 @@ pipeline {
             }
           }
         }
+        stage('Run E2E Tests for Packages'){
+          agent { label 'ubuntu && immutable' }
```

Comment:
```suggestion
          agent { label 'ubuntu-18 && immutable' }
```

Reply:
Can we apply this to master and then backport?

Reply:
you have to use fixed Ubuntu workers version, it not it will run on 14.04, 16.04, 18.04, or 20.04 with a random result.